modules/post/android/local/koffee.rb

```diff
@@ -0,0 +1,241 @@
+# frozen_string_literal: true
+
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+require 'readline'
+
+class MetasploitModule < Msf::Post
+  Rank = ExcellentRanking
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'KOFFEE - Kia OFFensivE Exploit',
+        'Description' => %q{
+          This module exploits the CVE-2020-8539, which is an Arbitrary Code Execution vulnerabilty that allows an to attacker execute the micomd binary file Kia Motors of Head Unit.
+          This module has been tested on SOP.003.30.18.0703, SOP.005.7.181019 and SOP.007.1.191209 head unit software version.
+          This module, run on an active session, allows an attacker to send crafted micomd commands that allow the attacker
+          to control the head unit and send CAN bus frames into the Multimedia CAN (M-Can) of the vehicle.
+        },
+        'Author' =>
+          [
+            'Gianpiero Costantino',
+            'Ilaria Matteucci'
+          ],
+        'References' =>
+          [
+            ['CVE', '2020-8539'],
+            ['URL', 'https://sowhat.iit.cnr.it/pdf/IIT-20-2020.pdf']
+          ],
+        'Platform' => 'Android',
+        'Arch' => [ARCH_ARMLE],
```

Comment:
Because you're not dealing with payloads, I don't think you need to define the `Arch`.

Reply:
Removed Arch. Even here by default when running the module, metasploit shows me the Rank tag as blank. If this is ok we can resolve this point